projects/api/src/lib/models/app-layer-with-service.model.ts

```diff
@@ -0,0 +1,6 @@
+import { AppLayerModel } from './app-layer.model';
+import { ServiceModel } from './service.model';
+
+export interface AppLayerWithServiceModel extends AppLayerModel {
```

Comment:
Just a suggestion: maybe its good to move this file to `core/src/lib/map/models`. So far the api folder contains mainly models returned by the backend api. This is model is "created" by selectors in map folder so might make more sense there.